src/erf.jl

```diff
@@ -281,3 +281,31 @@ function erfcinv(y::Float32)
 end
 
 erfcinv(x::Integer) = erfcinv(float(x))
+
+# MPFR has an open TODO item for this function
+# until then, we use [DLMF 7.12.1](https://dlmf.nist.gov/7.12.1) for the tail
+function erfcx(x::BigFloat)
```

Comment:
Shouldn't it work (with minor modifications) for `Complex{BigFloat}` too?   I guess we don't have a complex `erfc` function for bigfloat, though.

Reply:
Not in its current form, but it probably could be adapted.